.platform/confighooks/prebuild/01_assets_precompile.sh

```diff
@@ -2,16 +2,6 @@
 echo "Bundle install and yarn install"
 su webapp -c "yarn --production";
 
-echo "Performing permissions work"
-# chown -R webapp:webapp .bundle; 
-chmod -R 777 .bundle;
-# chown -R webapp:webapp vendor; 
-chmod -R 777 vendor;
-
 echo "Permission for home/webapp"
 mkdir -p /home/webapp
-chmod 700 /home/webapp;
-
-
-echo "Making environment variables available"
-export $(cat /opt/elasticbeanstalk/deployment/env | xargs)
+chmod 700 /home/webapp;
```

Comment:
Made this align with this file https://github.com/BarnesFoundation/Barnes-Focus/pull/44/files#diff-a8eebca460e0a3db23a0e98e15df4f1536354e78a39c8f9ff43958c2ec846995R1-R7.